src/parser/grammar.y

```diff
@@ -87,10 +87,28 @@ expr(A) ::= unwindClause(B) returnClause(C) skipClause(D) limitClause(E). {
 	A = New_AST_Query(NULL, NULL, NULL, NULL, NULL, NULL, C, NULL, D, E, NULL, B);
 }
 
-%type matchClause { AST_MatchNode* }
+%type multipleMatchClause { AST_MatchNode* }
```

Comment:
I think naming this type `matchClauses` would keep our expression format more standardized (the `matchClauses` could be `matchClauseChains` or something like that). It's not an issue, though, so no need to change if you disagree!

Reply:
The multipleMatchClauses has the exact same structure as multipleCreateClauses
if our intention is to move a way from our own parsing, something UNWIND will probably enforce us to then I can live with this.